packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mirrorworld/mirage.core",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
In the future, there is no need to manually change the version. Just create the changeset and it will be okay. Our CI/CD will automatically version the packages according to the changeset and publish the changes when merged to the `main` branch.

Reply:
Ok nice, updated it accordingly